packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -964,7 +964,7 @@ export class FluidDataStoreRuntime
 	 * @param content - Content of the signal.
```

Comment:
Shall we comment on any of the others about content needing to be JSON serializable?
(BTW doesn't have to be an `object`, could be `string`, `number`, or `boolean` as well. Could say "JSON serializable object or primitive".)